thirdeye-persistence/src/main/resources/db/migration/mysql/V1_307_0__add_namespace_configuration_entity_table.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS namespace_configuration_entity
+(
+    id                             bigint(20) PRIMARY KEY AUTO_INCREMENT,
+    namespace                      varchar(200),
+    timezone                       varchar(100) NOT NULL,
```

Comment:
if we don't use JOOQ, is there a use case for flattening timezone, date_time_pattern, minimum_onboarding_start_time? 
we usually flatten to add indexes (btw it's not really required anymore in MySQL 8, we could index json fields directly I think)

```
id
namespace
create_time
update_time
version
```

my understanding is the only query patterns will be: 
```
select where namespace= 
update where namespace = 
select where id=
update where id=
```

Reply:
sorry, i don't understand this
are you suggesting to have an inner table called TimeConfiguration and add that as a column instead of individual fields directly? if yes, do we have any such example that i can refer?

Reply:
nvm, understood 
Removed flatten out fields since they're not needed for now